org.geeqie.Geeqie.json

```diff
@@ -0,0 +1,121 @@
+{
+    "app-id": "org.geeqie.Geeqie",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
```

Comment:
Any reason for not using 3.34?

Reply:
None.
After a 5min web search, I gave up on finding out what the most recent one is and was hoping you would tell me if it would be outdated 😇